### PR TITLE
randomise loading messages

### PR DESCRIPTION
**Pull request by stojg**

### CODE REVIEW

**Comment by stojg** [comment]
supersede https://github.com/silverstripe/deploynaut/pull/857/

File: js/containers/buttons/SaveTargetRelease.jsx [hunk 1]
```diff
@@ -2,9 +2,13 @@ var ReactRedux = require('react-redux');
...
 var actions = require('../../_actions.js');
 var Button = require('../../components/Button.jsx');
+var message = require('../../constants/messages.js');
+
+// keep it the same between render calls by deciding the random message on import
+const loadingMessage = message.getRandom();
 
 const mapStateToProps = function(state) {
-	const text = state.plan.is_loading ? 'Reticulating splines' : 'Continue';
+	const text = state.plan.is_loading ?  loadingMessage : 'Continue';
```
Comment (halkyon) [inline js/containers/buttons/SaveTargetRelease.jsx]: SUPER CRITICAL ISSUE. ` loadingMessage` instead of `loadingMessage`

Comment (stojg) [inline js/containers/buttons/SaveTargetRelease.jsx]: wut?

**Comment by halkyon** [comment]
could you also do this for "Requesting abort" on the abort button? :)

File: js/containers/buttons/SaveTargetRelease.jsx (hunk 1 above)
Comment (stojg) [inline js/containers/buttons/SaveTargetRelease.jsx]: ah, ws

**Comment by stojg** [comment]
I tried adding it to the abort button, but it makes less sense since it's a "momentary" button that then goes back to the 'Abort' state. I also don't want to joke around with abort, since the user is probably a bit stressed at that point.

**Pull request merged**